spec/models/uploaded_file_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe UploadedFile, type: :model do
+
+  describe "accepted content types" do
+    it "png" do
+      expect(build(:uploaded_file, :png)).to be_valid
+    end
+    it "gif" do
+      expect(build(:uploaded_file, :gif)).to be_valid
+    end
+    it "jpg" do
+      expect(build(:uploaded_file, :jpg)).to be_valid
+    end
+
+    it "pdf" do
+      expect(build(:uploaded_file, :pdf)).to be_valid
+    end
+    it "txt" do
+      expect(build(:uploaded_file, :txt)).to be_valid
+    end
```

Comment:
There is no type for ".doc" files above.

Also, probably need to add support for ".docx" types.